Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -118,22 +138,38 @@ open class NavigationMapView: UIView {
         removeRoutes()
         removeWaypoints()
         
-        show(routes)
+        switch routesPresentationStyle {
+        case .single:
+            show([activeRoute])
+        case .all(_):
+            show(routes)
+        }
+        
         showWaypoints(on: activeRoute)
         
         navigationCamera.stop()
-        fitCamera(to: activeRoute, animated: animated)
+        fitCamera(to: routes,
+                  routesPresentationStyle: routesPresentationStyle,
+                  animated: animated)
     }
     
     /**
      Visualizes the given routes, removing any existing routes from the map.
      
-     Each route is visualized as a line. Each line is color-coded by traffic congestion, if congestion levels are present. Implement `NavigationMapViewDelegate` methods to customize the appearance of the lines representing the routes. To also visualize waypoints and zoom the map to fit, use the `showcase(_:animated:)` method.
+     Each route is visualized as a line. Each line is color-coded by traffic congestion, if congestion
+     levels are present. Implement `NavigationMapViewDelegate` methods to customize the appearance of
+     the lines representing the routes. To also visualize waypoints and zoom the map to fit,
+     use the `showcase(_:animated:)` method.
      
      To undo the effects of this method, use the `removeRoutes()` method.
      
-     - parameter routes: The routes to visualize in order of priority. The first route is displayed as if it is currently selected or active, while the remaining routes are displayed as if they are currently deselected or inactive. The order of routes in this array may differ from the order in the original `RouteResponse`, for example in response to a user selecting a preferred route.
-     - parameter legIndex: The zero-based index of the currently active leg along the active route. The active leg is highlighted more prominently than inactive legs.
+     - parameter routes: The routes to visualize in order of priority. The first route is displayed
+     as if it is currently selected or active, while the remaining routes are displayed as if they
+     are currently deselected or inactive. The order of routes in this array may differ from the
+     order in the original `RouteResponse`, for example in response to a user selecting a preferred
+     route.
+     - parameter legIndex: The zero-based index of the currently active leg along the active route.
+     The active leg is highlighted more prominently than inactive legs.
```

Comment:
I took an opportunity to reformat comments in `NavigationMapView` to be a bit shorter. It's pretty inconvenient to use horizontal scrolling while reading them.

Reply:
I'm with @MaximAlien here. It much easier to read documentation that is wrapped in a reasonable column width (80-120 is most commonly used). Besides this there are other advantages:
- Easier in read on GitHub for Web.
- Works without wrapping setting in the editor.
- Human can wrap text better. 
- Anything else? I've seen many other repos doing the same thing (including the ones from Apple).

Reply:
This isn’t something I feel strongly about; I just pointed out the downsides for awareness. In my opinion, the fact that GitHub’s file viewing mode doesn’t wrap lines is a sufficient argument in favor of hard wrapping. I’m just less convinced about the other points, since generally, we can assume most developers will use Xcode’s default (which is to soft-wrap).

Reply:
Even with wraps, the downside is that the text becomes hard to read because the text is too long[^1].

There are downsides as well as @1ec5 mentioned. But I think we should prioritize the decisions that make the lives of the devs who use our SDK easier/better. 

[^1]: [Readability of the text](https://ux.stackexchange.com/questions/108801/what-is-the-best-number-of-paragraph-width-for-readability)